cmd/cloudinfo/config.go

```diff
@@ -53,20 +70,27 @@ func defineFlags(pf *pflag.FlagSet) {
 func Configure(v *viper.Viper, pf *pflag.FlagSet) {
 	// configure viper
 	// Viper check for an environment variable
+
+	v.SetEnvKeyReplacer(strings.NewReplacer(".", "_", "-", "_"))
+
 	v.AutomaticEnv()
-	v.SetEnvKeyReplacer(strings.NewReplacer("-", "_"))
+
+	// Log configuration
+	v.RegisterAlias("log.format", "log-format")
+	v.RegisterAlias("log.level", "log-level")
+	v.RegisterAlias("log.noColor", "no_color")
 
 	pf.Init(FriendlyServiceName, pflag.ExitOnError)
 
 	// define flags
 	defineFlags(pf)
 
-	// parse the command line
-	pflag.Parse()
-
 	// bind flags to viper
 	if err := viper.BindPFlags(pf); err != nil {
 		panic(fmt.Errorf("could not parse flags. error: %s", err))
 	}
 
+	// parse the command line
+	pflag.Parse()
```

Comment:
This is a side effect, it should be in `main`

Reply:
thx, fixed